boards/catie/zest_core_stm32h753zi/zest_core_stm32h753zi.yaml

```diff
@@ -0,0 +1,19 @@
+identifier: zest_core_stm32h753zi
+name: Zest Core STM32H753ZI
```

Comment:
```suggestion
name: Zest_Core_STM32H753ZI
```

Reply:
Resolve in [453b73f](https://github.com/catie-aq/zephyr_zest-core-stm32h753zi/pull/26/commits/453b73ffe83643933775c6dad0af736bf7d96da8).